mlflow/transformers.py

```diff
@@ -70,6 +71,8 @@
 _MODEL_PATH_OR_NAME_KEY = "source_model_name"
 _SUPPORTED_SAVE_KEYS = {_MODEL_KEY, _TOKENIZER_KEY, _FEATURE_EXTRACTOR_KEY, _IMAGE_PROCESSOR_KEY}
 _SUPPORTED_RETURN_TYPES = {"pipeline", "components"}
+_TRANSFORMERS_DEFAULT_CPU_DEVICE_ID = -1
```

Comment:
Can you provide an inline comment / reference for this value? (i.e. somewhere where it states CPU == -1)

Reply:
Done!